airflow-core/src/airflow/cli/commands/db_manager_command.py

```diff
@@ -18,33 +18,50 @@
 
 from airflow import settings
 from airflow.cli.commands.db_command import run_db_downgrade_command, run_db_migrate_command
-from airflow.providers.fab.auth_manager.models.db import _REVISION_HEADS_MAP, FABDBManager
+from airflow.configuration import conf
 from airflow.utils import cli as cli_utils
+from airflow.utils.module_loading import import_string
 from airflow.utils.providers_configuration_loader import providers_configuration_loaded
 
 
+def _get_db_manager(name):
+    """Import the db manager class."""
+    db_managers = conf.get("database", "external_db_managers")
+    db_manager = None
+    for manager in db_managers.split(","):
+        if name in manager:
```

Comment:
Not sure this is right - we want a complete match right, not just a partial match? If we have a `a` and a `abc` we could match `a` to `abc`. Since we don't have the concept of a "name", maybe we should take the whole classpath?